sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -564,6 +595,7 @@ SIREPO.app.directive('srTooltip', function(appState, mathRendering, $interpolate
                         // evaluate angular text first if {{ }} is present
                         if (/\{\{.*?\}\}/.test(res)) {
                             $scope.appState = appState;
+                            $scope.SIREPO = SIREPO;
```

Comment:
Is this needed?

Reply:
This or some other way to get the schema info on the scope so that the interpolation works. This was quickest